regcomp.c

```diff
@@ -16329,7 +16326,7 @@ S_handle_names_wildcard(pTHX_ const char * wname, /* wildcard name to match */
      * were returned to us in the array 'algorithmic_names' from data in
      * lib/unicore/Name.pm.  'code_point' in the name is expressed in hex. */
     for (i = 0; i <= av_top_index((AV *) algorithmic_names); i++) {
-        IV j;
+        SSize_t j;
```

Comment:
If I understand the code `low` and `high` represent a range of code points, and so hence `j` is also a code point.

What I'm not sure about is what controls the Perl extended valid range of code points, if it's limited to non-negative `IV` values this change is incorrect, since IV/UV can be larger than `SSize_t`.

It might be that the range here is always within the base unicode range, but I'm not sure we limit custom properties to that (or if those apply here.)

Reply:
You're right